File: api-gateway/src/main/java/apiservice/car/model/Service.java
package apiservice.car.model;

import lombok.Data;

import java.time.ZonedDateTime;

@Data
public class Service {
    private int ServiceId;
    private String Origin;
    private String Destination;
    private int ClientId;
    private int DriverId;
    private String Description;
    private ZonedDateTime ServiceDatetime;
    private String CalendarEvent;
    private ZonedDateTime CalendarDatetime;
    private ZonedDateTime PayedDatetime;
    private Float BasePrice;
    private Float ExtraPrice;
    private ZonedDateTime ConfirmedDatetime;
    private int Passengers;
    private String SpecialNeeds;
}

File: legal/src/main/kotlin/com/car/rental/legal/model/Service.kt
package com.car.rental.legal.model

data class Service(
    val serviceId: Int,
    val origin: String?,
    val destination: String?,
    val clientId: Int,
    val driverId: Int,
    val description: String?,
    val serviceDatetime: String?,
    val calendarEvent: String?,
    val payedDatetime: String?,
    val basePrice: Float?,
    val extraPrice: Float?,
    val confirmedDatetime: String?,
    val passengers: Int,
    val specialNeeds: String?
)

